<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>POI_VisibilityWidget</class>
 <widget class="QDockWidget" name="POI_VisibilityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>110</height>
   </rect>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>兴趣点可见性</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
    <item row="0" column="2">
     <widget class="QToolButton" name="mHidePOI_Button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>隐藏</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>兴趣点</string>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="mShowPOI_Button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>显示</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>靶心</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="mShowBullseyeButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>显示</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="mHideBullseyeButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>隐藏</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
